test/fixtures/build-compiler/package.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This is just used as a minifier and is just a flag, so I'm not sure a new fixture is necessary, it could be used with the default fixture. The other fixture directories are for different configurations.

If configuring Closure, could use the `build-withConfig` directory since it's for testing `tsdx.config.js` (though it's not currently set-up for automated tests -- that needs to be fixed)

Reply:
done